code generators/de.fhdo.lemma.model_processing.code_generation.container_base/src/main/java/de/fhdo/lemma/model_processing/code_generation/container_base/ContainerBaseLanguageDescriptionProvider.xtend

```diff
@@ -0,0 +1,36 @@
+package de.fhdo.lemma.model_processing.code_generation.container_base
+
+import de.fhdo.lemma.model_processing.annotations.LanguageDescriptionProvider
+import de.fhdo.lemma.model_processing.languages.LanguageDescriptionProviderI
+import de.fhdo.lemma.operation.OperationPackage
+import de.fhdo.lemma.model_processing.languages.XtextLanguageDescription
+import de.fhdo.lemma.operationdsl.OperationDslStandaloneSetup
+import de.fhdo.lemma.operation.intermediate.IntermediatePackage
+import de.fhdo.lemma.model_processing.languages.LanguageDescription
+import org.eclipse.xtext.util.internal.Nullable
+
+/**
+ * This class contains the language description for LEMMA's modeling languages.
+ *
+ * @author [Philip Wizenty](mailto:philip.wizenty@fh-dortmund.de)
```

Comment:
Ditto. Please use the common Java syntax, which you also used for the operation intermediate model, i.e., `@author <a href="mailto:philip.wizenty@fh-dortmund.de">Philip Wizenty</a>`